specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/examples/Alerts/GetAlertsResourceGroupWithAutoDismissRule_example.json

```diff
@@ -0,0 +1,85 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "resourceGroupName": "myRg1",
+    "autoDismissAlertsRuleName": "dismissThreatIntelligence"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg1/providers/Microsoft.Security/locations/westeurope/alerts/2518770965529163669_F144EE95-A3E5-42DA-A279-967D115809AA",
+            "name": "2518770965529163669_F144EE95-A3E5-42DA-A279-967D115809AA",
+            "type": "Microsoft.Security/Locations/alerts",
+            "properties": {
+              "vendorName": "Microsoft",
+              "alertDisplayName": "Threat Intelligence Alert",
+              "alertName": "ThreatIntelligence",
+              "detectedTimeUtc": "2018-05-01T19:50:47.083633Z",
+              "description": "Process was detected running on the host and is considered to be suspicious, verify that the user run it",
+              "remediationSteps": "verify that the user invoked this process\r\nrun antimalware scan of the VM",
+              "actionTaken": "Detected",
+              "reportedSeverity": "High",
+              "compromisedEntity": "vm1",
+              "associatedResource": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg1/providers/Microsoft.Compute/virtualMachines/vm1",
```

Comment:
>20ff7fc3-e762-44dd-bd96-b71116dcdc23 [](start = 52, length = 36)

please verify that the subscription id in the examples is aligned with other examples of Microsoft.Security APIs

(I'm not sure if it is actually the subscription used to simulate alerts and generate the SDK libraries or it is a fake guid..)

Reply:
yes it is in most API examples